bitnami/concourse/templates/web/rbac.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.web.enable }}
+{{- if .Values.rbac.create -}}
+---
+apiVersion: rbac.authorization.k8s.io/{{ .Values.rbac.apiVersion }}
+kind: ClusterRole
+metadata:
+  name: {{ template "common.names.fullname" . }}
```

Comment:
If it's for the web, then it makes sense to use the -web or the concourse.web.fullname macro